k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -321,19 +321,17 @@ function deleteSecrets()
     printf "########################################################\n"
     printf "# Delete Secrets                                       #\n"
     printf "########################################################\n"
-    output=$(./parse_scripts/parse_yaml.sh $solution_yaml "solution.secrets*.name")
-    IFS=';' read -r -a parsed_secret_name_array <<< "$output"
-    for secret_name in "${parsed_secret_name_array[@]}"
-    do
-        secret_name=$(echo $secret_name | cut -f2 -d'>')
+    secret_name=$(./parse_scripts/parse_yaml.sh $solution_yaml "solution.secrets.name")
+    if [[ ! -z "${secret_name}" ]]; then
+        secret_fname=$(echo "${secret_name}" | cut -f2 -d'>')
         kubectl delete secret $secret_name --namespace=$namespace
```

Comment:
I think you need this to be `${secret_fname}` instead of `$secret_name`.

```bash
########################################################
# Delete Secrets                                       #
########################################################
Error from server (NotFound): secrets "solution.secrets.name>cortx-secret" not found
```
```suggestion
        kubectl delete secret ${secret_fname} --namespace=$namespace
```

Reply:
Aarg! I can't believe I missed that.  Good catch.

Reply:
Done.